web/src/routes/SearchPage.tsx

```diff
@@ -82,7 +82,16 @@ const SearchPage = ({ city, cityCode, languageCode, pathname }: CityRouteProps):
     />
   )
 
-  if (loading || !results) {
+  if (query.length === 0) {
+    return (
+      <CityContentLayout isLoading {...locationLayoutParams}>
+        <Helmet pageTitle={pageTitle} languageChangePaths={languageChangePaths} cityModel={city} />
+        {SearchBar}
+      </CityContentLayout>
+    )
+  }
+
+  if (loading) {
```

Comment:
:wrench: How about deleting both these two blocks (`if (query.length === 0)` and `if (loading)`) and just adding the suggestion below? i think the behavior should be the (nearly the) same with a lot less code. Also, it avoids not showing an error just because the query is empty.
The only difference is that results will be displayed faster even though not all endpoints have loaded yet.